pkg/bff/api/v1/api.go

```diff
@@ -104,6 +104,7 @@ type RegisterReply struct {
 
 // OverviewReply is returned on overview requests.
 type OverviewReply struct {
+	Error   NetworkError    `json:"error,omitempty"`
 	OrgID   string          `json:"org_id"`
 	TestNet NetworkOverview `json:"testnet"`
 	MainNet NetworkOverview `json:"mainnet"`
```

Comment:
The frontend should be able to check the error field to determine if the summary requests failed for testnet and/or mainnet.